code/modules/shuttles/shuttle_supply.dm

```diff
@@ -23,7 +23,7 @@
 
 /datum/shuttle/ferry/supply/proc/pick_loc()
 	RETURN_TYPE(/turf)
-	return GLOB.supply_elevator
+	return GLOB.supply_elevator_turfs[1]
```

Comment:
```suggestion
    if(length(GLOB.supply_elevator_turfs))
		return GLOB.supply_elevator_turfs[1]
```
check length of the list here - if it doesn't exist, trying to access the first one in the index will error (like in CI where there's no supply elevator for the groundmaps)